### DISCUSSION TITLE
Where is the C++ SDK?

### DISCUSSION THREAD

**Discussion by lespalt** [elgatosf/elgato#25] [General] [answered]
I've been searching for the C++ SDK for a while now, but can't find it on either Elgato's website or github.  The "streamdeck-cpu" sample mentioned at https://developer.elgato.com/documentation/stream-deck/samples/cpu/#source-code references files like StreamDeckSDK/ESDConnectionManager.h, but I can't find the official source for them. What am I missing?  Thanks

**Comment by GeekyEggo** [comment, marked as answer]
Hey @lespalt, whilst we don't provide an official C++ SDK at this time, the wonderful Fred Emmott does, in the form of the [StreamDeck-CPPSDK](https://github.com/fredemmott/StreamDeck-CPPSDK) library.  I don't have much experience with the cpu sample plugin, but I believe what may have started as a partial SDK in the cpu plugin has now been inherited and extended by Fred, who now maintains a community lead C++ SDK.
